xpcPlugin/MessageHandlers.cpp

```diff
@@ -26,6 +26,9 @@
 #include <cmath>
 #include <cstring>
 
+#define MULTICAST_GROUP "239.255.1.1"
```

Comment:
Where does this value come from? Is it a good default value that should be baked in? If there was some non-arbitrary reason for this value, let's mention it in a comment

Reply:
I used the same group (but a different port) as the native X-Plane BECN packet. See this comment https://github.com/nasa/XPlaneConnect/issues/147#issuecomment-446257549 I believe we are free to choose whatever arbitrary multicast group.